samples/ComputeSharp.SwapChain/Program.cs

```diff
@@ -35,28 +35,35 @@ class Program
 
     static void Main()
     {
-        Console.WriteLine("Available samples:");
-        Console.WriteLine();
+        int index;
 
-        for (int i = 0; i < Samples.Length; i++)
+        do
         {
-            Console.WriteLine($"{i}: {Samples[i].GetType().GenericTypeArguments[0].Name}");
-        }
+            Console.WriteLine("Available samples:");
```

Comment:
If we want to re-print these, we should clear the console first so that it looks cleaner 🙂
```suggestion
            Console.Clear();
            Console.WriteLine("Available samples:");
```